refkit-init-build-env

```diff
@@ -30,50 +30,23 @@ if [ -n "$BASH_SOURCE" ]; then
 elif [ -n "$ZSH_NAME" ]; then
     THIS_SCRIPT=$0
 else
-    THIS_SCRIPT="$(pwd)/oe-init-build-env"
-fi
-if [ -n "$BBSERVER" ]; then
-    unset BBSERVER
+    THIS_SCRIPT="$(pwd)/refkit-init-build-env"
 fi
 
-if [ -z "$ZSH_NAME" ] && [ "$0" = "$THIS_SCRIPT" ]; then
-    echo "Error: This script needs to be sourced. Please run as '. $THIS_SCRIPT'"
-    exit 1
-fi
+THIS_SCRIPT_DIR=$(dirname "$THIS_SCRIPT")
+THIS_SCRIPT_DIR=$(readlink -f "$THIS_SCRIPT_DIR")
 
-if [ -z "$TEMPLATECONF" ]; then
-    THIS_SCRIPT_DIR=$(dirname "$THIS_SCRIPT")
-    THIS_SCRIPT_DIR=$(readlink -f "$THIS_SCRIPT_DIR")
-    if [ -f "$THIS_SCRIPT_DIR/.templateconf" ]; then
-        source $THIS_SCRIPT_DIR/.templateconf
-    elif [ -d "$THIS_SCRIPT_DIR/.template" ]; then
-        TEMPLATECONF="$THIS_SCRIPT_DIR/.template"
-    fi
+if [ -z "$OEROOT" ]; then
+    OEROOT="$THIS_SCRIPT_DIR"/openembedded-core
 fi
```

Comment:
This patch on top of your tree fixes the problem:

```
diff --git a/.templateconf b/.templateconf
deleted file mode 100644
index ef125d2642..0000000000
--- a/.templateconf
+++ /dev/null
@@ -1,2 +0,0 @@
-# Template settings (relative to OEROOT)
-TEMPLATECONF=${TEMPLATECONF:-../meta-refkit/conf}
diff --git a/refkit-init-build-env b/refkit-init-build-env
index e76c26fe5c..a9a8d685f8 100755
--- a/refkit-init-build-env
+++ b/refkit-init-build-env
@@ -44,7 +44,8 @@ if [ -z "$BITBAKEDIR" ]; then
     BITBAKEDIR="$THIS_SCRIPT_DIR"/bitbake
 fi
 
-. "$THIS_SCRIPT_DIR"/.templateconf
+# Template settings (relative to OEROOT)
+TEMPLATECONF=${TEMPLATECONF:-"$THIS_SCRIPT_DIR"/meta-refkit/conf}
 
 unset THIS_SCRIPT
 unset THIS_SCRIPT_DIR
```

Note that the separate .templateconf file doesn't seem to be needed at all, so I'm removing it.

I also wonder whether should have a oe-init-build-env -> refkit-init-build-env symlink or even rename our file entirely. There's no reason for the script to be named differently, it's just another learning step for people trying to get started.